src/app/users/users-update/users-update.component.html

```diff
@@ -47,13 +47,14 @@
           <mat-datepicker #dp></mat-datepicker>
           <mat-error><planet-form-error-messages [control]="editForm.controls.birthDate"></planet-form-error-messages></mat-error>
         </mat-form-field>
-        <mat-form-field>
-          <mat-select i18n-placeholder placeholder="Gender" formControlName="gender" required>
-            <mat-option value="male">Male</mat-option>
-            <mat-option value="female">Female</mat-option>
-          </mat-select>
-          <mat-error><planet-form-error-messages [control]="editForm.controls.gender"></planet-form-error-messages></mat-error>
-        </mat-form-field>
+        <div>
+          <mat-radio-group i18n-placeholder formControlName="gender" required>
+            <label>Gender</label>
+            <mat-radio-button class="planet-radio-button" value="male"><mat-icon svgIcon="male" class="male-icon primary-text-color"></mat-icon>Male</mat-radio-button>
```

Comment:
Both `mat-ratio-button` elements require an i18n attribute to tag "Male" and "Female" for translation.